test/es-module/test-esm-exports.mjs

```diff
@@ -27,20 +30,40 @@ import { requireFixture, importFixture } from '../fixtures/pkgexports.mjs';
       }));
   }
```

Comment:
can we add a test for `./node_modules/inner/lib.js`; regardless of if it errors or is allowed, just making sure we don't break it in the future.

Reply:
Let's discuss this some more first.